HerPublicWebsite.ManagementShell/CommandHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace HerPublicWebsite.ManagementShell;
+
+public class CommandHandler
+{
+    private readonly IDatabaseOperation databaseOperation;
+    private readonly IFakeReferralGenerator fakeReferralGenerator;
+    private readonly IOutputProvider outputProvider;
+
+    public CommandHandler(IDatabaseOperation databaseOperation, IFakeReferralGenerator fakeReferralGenerator,
+        IOutputProvider outputProvider)
+    {
+        this.databaseOperation = databaseOperation;
+        this.fakeReferralGenerator = fakeReferralGenerator;
+        this.outputProvider = outputProvider;
+    }
+
+    public void GenerateReferrals(string[] args)
+    {
+        if (args.Length == 0)
+        {
+            outputProvider.Output("Missing number of referrals. Usage: 'GenerateReferrals <count>'.");
+            return;
+        }
+
+        if (!int.TryParse(args[0], out var referralCount))
+            outputProvider.Output("Invalid number of referrals. Usage: 'GenerateReferrals <count>'.");
+
+        outputProvider.Output("!!!!!!!!!!!!!!!!!!!!!!");
+        outputProvider.Output(
+            $"You are about to generate {referralCount} fake referral requests and add them to the database.");
+        outputProvider.Output("This command should only be run on development and staging environments.");
+        outputProvider.Output("Double check that you are NOT running this command on a production environment.");
+        outputProvider.Output("All users will have a FullName that begins \"FAKE USER\"");
```

Comment:
Want to make it really easy to undo: if someone does run this somewhere they shouldn't then they will probably panic. Could you add something like:
```
To revert, connect to the database and run the following:
DELETE FROM "users" WHERE "name" LIKE 'FAKE USER%' -- and created_at = today, if that's possible?
```

Reply:
done, though to note there the `RequestDate` field is faked & so it's not really possible to know when the referrals were generated, unless we'd like to change this field

Reply:
Should be fine - not a problem to delete them all, given you can always add new ones.